js/navigation.js

```diff
@@ -242,12 +248,14 @@ bcpl.navigation = (($, keyCodes) => {
 		}
 	};
 
-	$(document).on('mouseover', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseover);
-	$(document).on('mouseleave', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseleave);
-	$(document).on('keydown', '#responsive-sliding-navigation button', navigationButtonKeyPressed);
-	$(document).on('keydown', '#responsive-sliding-navigation', navigationKeyPressed);
-	$(document).on('click', navButtonSelector, navButtonClicked);
-	$(document).on('keydown', '#responsive-sliding-navigation a', navigationMenuItemKeyPressed);
+	$(document)
+		.on('mouseover', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseover)
+		.on('mouseleave', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseleave)
```

Comment:
The selector here is fairly massive and is repeated.  Can this move out to a `const` like `navButtonSelector` and `backButtonSelector`?

Reply:
yea it probably should, but this wasn't a refactor.